xbmc/utils/test/TestURIUtils.cpp

```diff
@@ -358,11 +353,6 @@ TEST_F(TestURIUtils, IsOnDVD)
 
 TEST_F(TestURIUtils, IsOnLAN)
 {
-  std::vector<std::string> multiVec;
-  multiVec.push_back("daap://path/to/file");
-  EXPECT_TRUE(URIUtils::IsOnLAN(CMultiPathDirectory::ConstructMultiPath(multiVec)));
-  EXPECT_TRUE(URIUtils::IsOnLAN("stack://daap://path/to/file"));
```

Comment:
these tests arent testing anything related to daap as such. make it smb:// instead.